build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.8.3-2
+v1.9.0rc1-1
```

Comment:
yes, it is intended. Seems like we've consistently used `vX.Y.Z-N` here, so this probably sorts/lists better